app/build.gradle

```diff
@@ -24,7 +24,7 @@ android {
     }
 
     // We have a three dimensional build configuration:
-    // BUILD TYPES (debug, beta, release) X PRODUCT FLAVOR (focus, klar) X ENGINE FLAVOR (webkit, gecko(
+    // BUILD TYPES (debug, beta, release) X PRODUCT FLAVORS (focus, klar) X ENGINE FLAVORS (webkit, gecko)
```

Comment:
The typo is actually the "S" in "BUILD TYPES". This was meant to be a cross product: build type X product flavor X engine flavor.